libvcx/src/aries/handlers/proof_presentation/prover/state_machine.rs

```diff
@@ -292,22 +292,22 @@ pub mod test {
 
     impl ProverSM {
         fn to_presentation_prepared_state(mut self) -> ProverSM {
-            self = self.step(ProverMessages::PreparePresentation((_credentials(), _self_attested())), None).unwrap();
+            self = self.step(ProverMessages::PreparePresentation((_credentials(), _self_attested())), None::<&Box<dyn Fn(&A2AMessage) -> VcxResult<()>>>).unwrap();
```

Comment:
Let's just make sure the typing of `None` in the tests stays consistent with my previous comment.